mlrun/common/schemas/notification.py

```diff
@@ -42,13 +42,13 @@ class NotificationStatus(mlrun.common.types.StrEnum):
 
 
 class Notification(pydantic.BaseModel):
-    kind: NotificationKind = None
-    name: str = None
-    message: str = None
-    severity: NotificationSeverity = None
-    when: typing.List[str] = None
-    condition: str = None
-    params: typing.Dict[str, typing.Any] = None
+    kind: NotificationKind
+    name: str
+    message: str
+    severity: NotificationSeverity
+    when: typing.List[str]
+    condition: str
+    params: typing.Dict[str, typing.Any] = {}
```

Comment:
that is a mutable default parameter.
In python, the mutable are computed once and each change is defined to all consecutive calls. default to None and enrich if is None.